netlify.toml

```diff
@@ -1,79 +1,97 @@
+    #   https://prefect--partial.my.salesforce.com/
+    #   https://prefect--partial.sandbox.my.salesforce.com/
+    # img-src
+    #   * - allow all images
+    #   data: - images embedded inline are permitted
+    # media-src
+    #   'none' - no audio or video files are permitted
+    # object-src
+    #   'none' - no legacy objects are allowed; see
+    #            https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/object-src
+    # script-src
+    #   'self' - all resources from current origin are permitted
+    # style-src
+    #   'self' - all styles from current origin are permitted
+    #   'unsafe-inline' - Styles are injected inline
+    # child-src
+    #   data: - web workers and frames embedding inline data
+    #   blob: - web workers and frames embedding inline data
+    # worker-src
+    #   'self' - all web workers from current origin are permitted
+    #   data: - web workers embedding inline data
+    #   blob: - web workers embedding inline data
+    Content-Security-Policy = """ \
       default-src *; \
+      frame-ancestors 'none'; \
       script-src 'self' 'unsafe-eval' 'unsafe-inline' https://cdn.lr-ingest.io https://js.stripe.com https://prefect.my.salesforce.com/ https://prefect.force.com/ https://prefect.my.salesforce.com https://c.la1-c2-ia2.salesforceliveagent.com/ https://d.la1-c2-ia2.salesforceliveagent.com/ https://static.lightning.force.com https://prefect--partial.my.salesforce.com https://d.la3-c1cs-ia2.salesforceliveagent.com/ https://service.force.com/ https://prefect--aleysian.my.salesforce.com https://aleysian-prefect.cs29.force.com/ https://d.la1-c2-ph2.salesforceliveagent.com/ https://partial-prefect.cs1.force.com/ https://c.la1-c1cs-ph2.salesforceliveagent.com/ https://d.la1-c1cs-ph2.salesforceliveagent.com/ https://prefect--partial.sandbox.my.salesforce.com/ https://d.la2-c1-ia5.salesforceliveagent.com https://d.la3-c1cs-ia5.salesforceliveagent.com https://prefect--partial.sandbox.my.site.com; \
-      script-src-elem 'self' 'unsafe-eval' 'unsafe-inline' https://cdn.lr-ingest.io https://js.stripe.com https://netlify-cdp-loader.netlify.app/netlify.js https://prefect.my.salesforce.com/ https://prefect.force.com/ https://prefect.my.salesforce.com https://c.la1-c2-ia2.salesforceliveagent.com/ https://d.la1-c2-ia2.salesforceliveagent.com/ https://static.lightning.force.com https://prefect--partial.my.salesforce.com https://d.la3-c1cs-ia2.salesforceliveagent.com/ https://service.force.com/ https://prefect--aleysian.my.salesforce.com https://aleysian-prefect.cs29.force.com/ https://d.la1-c2-ph2.salesforceliveagent.com/ https://partial-prefect.cs1.force.com/ https://c.la1-c1cs-ph2.salesforceliveagent.com/ https://d.la1-c1cs-ph2.salesforceliveagent.com/ https://prefect--partial.sandbox.my.salesforce.com/ https://d.la2-c1-ia5.salesforceliveagent.com https://d.la3-c1cs-ia5.salesforceliveagent.com https://prefect--partial.sandbox.my.site.com; \
```

Comment:
I don't think we can cut this (though we could potentially trim the list) - with the exception of Safari, all major browsers support the policy AND rely on it for inline handlers (they will not fall back to `script-src` afaict)